### ISSUE TITLE
[2026/3/4]推薦・機械学習勉強会

### ISSUE THREAD

**Issue by Hayashi-Yudai** [wantedly/machine-learning-round-table#335]
## Why

推薦・機械学習勉強会は、推薦や機械学習、その周辺技術を通じてサービスを改善することにモチベーションのある人達の集まりです。ニュースやブログから論文まで、気になったものについてお互い共有しましょう！

発信のため、ここは **public** にしてあります。外部からの参加をご希望の方は[林/python_walker](https://x.com/python_walker)、[市村/chimuichimu](https://x.com/chimuichimu1) まで DM を送るか、Wantedly Visit の募集（https://www.wantedly.com/projects/391912） よりご連絡ください！

## What

Wantedly では隔週水曜日に

- 推薦の評価指標について議論したい
- 〇〇っていうライブラリ / フレームワークを導入してみたい
- 他社の基盤事例をみんなにシェアして自社の基盤開発に活かしたい
- もっと推薦を良くするためにどんなものが必要か議論したい

といった話をする「推薦・機械学習勉強会」を開催しています。
この ISSUE はその会で話すネタを共有するための場所です。

話したいことがある人はここにコメントしましょう！
会の間に話した内容もここにメモしましょう！

prev: https://github.com/wantedly/machine-learning-round-table/issues/333

**Comment by Nsakdev** [comment]
## [HyTRec: A Hybrid Temporal-Aware Attention Architecture for Long Behavior Sequential Recommendation](https://arxiv.org/abs/2602.18283)

<img width="1174" height="336" alt="Image" src="https://github.com/user-attachments/assets/0a66f465-4c26-4b0e-bad7-2e4b22d4c39d" />

### 要点
- Hybrid Attention : ユーザ系列を直近 ($K$) 個の短期系列とそれ以前の長期系列に分解し、短期系列は精度重視でSoftmax Attention、長期系列は効率重視でLinear Attention + Softmax Attentionのハイブリッド
  - ハイブリッド比率を調整することで、推薦制度と遅延のトレードオフを取る
- Temporal-Aware Delta Net: どれだけ過去を信用するかで長期嗜好と短期スパイクをゲーティングしつつ、過去の情報を一定の強度で上書き更新

### 結果
- SOTAベースラインに対してHyTRecはどうか？
  - H@500、AUCで競争力のある結果。NDCG@500は悪化するケースあり。
- 超長系列までスケールしても学習・推論効率を保てるか？
  - 系列長が 100〜12k に伸びても、HyTRecはスループットが緩やかに低下するだけで、超長系列で効率劣化が急激に起きにくい
- 主要コンポーネント（TADNと短期系列ブランチ）はどれだけ効いているか？
  - 短期注意ブランチもTADNブランチもそれぞれ単体で有効で、両方を統合したフルモデルが最良
- ハイブリッド比率が精度とレイテンシにどう効くか？
  - 検証した混合比率（2:1〜6:1）の範囲では 3:1が性能と推論効率の最良トレードオフ
- コールドスタート等の厳しいケースにどれだけ頑健か？
  - 疎な履歴を持つユーザに対して、HyTRecはコールドスタート新規ユーザとサイレント旧ユーザの2ケースで良い性能を示し、頑健性がある（ただし比較なし）

**Comment by ghibney** [comment]
## [LLMでソート](https://joisino.hatenablog.com/entry/llmsort)

* LLM を比較関数として使うことで、論文・ニュース・政治スタンスなど 曖昧で主観的な基準でもソートできることを紹介している
* LLM は複数要素比較やノイズ・非推移性など 古典的ソートの前提と異なる性質を持つため、それに適した新しいソート設計が研究されている

**Comment by wakama1994** [comment]
### [APIファーストで作るMLOps基盤](https://zenn.dev/turing_motors/articles/71f1e96311885c)

- 前回発表できなかったTuring社のML基盤の事例

**Comment by Hayashi-Yudai** [comment]
## [ABEMAで取り組んだAIエージェントによるA/Bテスト文化の促進](https://developers.cyberagent.co.jp/blog/archives/62375/)

インターン生の取り組み。ABEMA では施策について以下のような課題があったらしい

- A/Bテストの分析が結果のシンプルな評価で終わってしまいがち
- 施策で得られた知見が属人化しがち

これに対してエージェントを用いたシステムを構築することで、知見の検索・活用をしやすい環境を作ったという話を紹介している。

**Comment by chimuichimu** [comment]
## [MediaFM: The Multimodal AI Foundation for Media Understanding at Netflix](https://netflixtechblog.com/mediafm-the-multimodal-ai-foundation-for-media-understanding-at-netflix-e8c28df82e2d)

### 概要

Netflixが映像・音声・字幕を統合し、ショット単位の汎用埋め込みを生成するFoundation Model「MediaFM」を開発。文脈化されたショット表現を一度生成し、広告配信やクリップ選定など複数の下流タスクで使い回す設計。

### 背景

- カタログ内コンテンツの機械的理解が推薦・プロモーション最適化に必要
- 長編映像の理解には前後のショットの文脈やマルチモーダル信号の活用が不可欠

### 入力表現

- ショット境界検出で分割し、各ショットから映像（SeqCLIP）・音声（wav2vec2）・字幕（text-embedding-3-large）の埋め込みを生成
- 3つを連結・正規化して2304次元のfused embeddingを構成

### モデルアーキテクチャ

- BERTライクなTransformerエンコーダで、ショット系列を文脈化
- Masked Shot Modeling（20%マスク → 元の埋め込みをコサイン距離で予測）で自己教師あり学習

### 評価タスクと結果

- 広告関連性・クリップ人気・トーン分類・ジャンル分類・クリップ検索の全タスクでベースラインを上回る
- アブレーションの結果、マルチモーダル統合より**文脈化が主要な改善要因**

**Comment by nogawanogawa** [comment]
## [HR領域の「言葉の壁」を越える。HRドメイン特化SPLADEモデル構築パイプラインの全貌](https://engineering.visional.inc/blog/728/hr-sparse-model-training-pipeline/)

HR検索（求人・レジュメ検索）では次の問題がある：
- HR特有の専門用語が多い
- 同義語・言い換えが多い
- 一般的な検索モデルではうまくマッチしない

このもんだいに対して、

- Phase1  Domain Pretraining
- Phase2  Knowledge Distillation
- Phase3  SPLADE Fine-tuning

で対応している。

<img width="1232" height="593" alt="Image" src="https://github.com/user-attachments/assets/7c8bbee7-3047-45fa-9486-3bd268fe7de8" />

モデルの学習については蒸留によって段階的に学習する

<img width="1232" height="318" alt="Image" src="https://github.com/user-attachments/assets/8f0bbafa-e069-40dd-8ca5-7c100cfeb8ff" />

1. Cross encoderの学習 (Teacher model)
2. 蒸留用類似度スコア（Soft Label）の生成
3.  SPLADE Fine-tuning
    - ここでsparse representationを作る (これができれば転置インデックスが作れる -> 高速に検索ができる)

splade（student モデル）についてはロスを2つ用意する。

<img width="1232" height="410" alt="Image" src="https://github.com/user-attachments/assets/c814a709-685e-4ae3-8a34-af332ee1406b" />

- 知識蒸留によるランキング学習
- スパース正則化による活性制御
    - 精度を上げようとするとモデルは自然にdenseにしてしまう（高速化の妨げ）のでそれに対してペナルティを与える

## [KDD Cup 2026](https://dataagent.top/)

DataAgent作成がテーマらしい
